_components/form-controls/04-date-input.md

```diff
@@ -2,12 +2,13 @@
 title: Date input
 parent: Form controls
 maturity: alpha
-order: 05
+order: 04
```

Comment:
I still see Data visualizations in this order:
![image](https://user-images.githubusercontent.com/3385219/87089169-9b057680-c1fb-11ea-979f-a8671de636c6.png)

On https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/uswds-2.8.0/components/form-controls/